test/minnum.ll

```diff
@@ -0,0 +1,39 @@
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv %t.bc -o %t.spv
+; RUN: spirv-val %t.spv
+; RUN: llvm-spirv --to-text %t.spv -o - | FileCheck %s --check-prefix=CHECK-SPIRV
+; RUN: llvm-spirv -r %t.spv -o %t.rev.bc
+; RUN: llvm-dis %t.rev.bc -o - | FileCheck %s --check-prefix=CHECK-LLVM
+
+
+target datalayout = "e-i64:64-v16:16-v24:32-v32:32-v48:64-v96:128-v192:256-v256:256-v512:512-v1024:1024"
+target triple = "spir64-unknown-unknown"
+
+; CHECK-SPIRV: Function
+; CHECK-SPIRV: FunctionParameter {{[0-9]+}} [[x:[0-9]+]]
+; CHECK-SPIRV: FunctionParameter {{[0-9]+}} [[y:[0-9]+]]
+; CHECK-SPIRV: ExtInst {{[0-9]+}} [[res:[0-9]+]] {{[0-9]+}} fmin [[x]] [[y]]
+; CHECK-SPIRV: ReturnValue [[res]]
+define spir_func float @TestHalf(float %x, float %y) {
+entry:
+  ; CHECK-LLVM: call spir_func float @[[TestHalfName:.+]](float %{{.+}}, float %{{.+}})
```

Comment:
What is this check suppose to validate?
I think we should look for exact name.

Reply:
Got it, I'll update this to match the exact mangling.